files-community/Files
# Fix: Fixed issue where directly opening library would invoke explorer.exe

**Resolved / Related Issues**
   Closes #issue...

Fixes issue #14302

**Validation**
How did you test these changes?
   1. Open app.
   2. Navigate to `~\AppData\Roaming\Microsoft\Windows\Libraries`.
   3. Open any of the `.library-ms` files.
